test/lit/fuzz-types/isorecursive.test

```diff
@@ -30,3 +30,37 @@
 ;; CHECK-NEXT:  (type $18 none)
 ;; CHECK-NEXT: )
 ;; CHECK-NEXT: (type $19 (func_subtype (param (ref any) i32 i64 f64 f64 anyref v128) (result i64) $9))
+;; CHECK-NEXT:
+;; CHECK-NEXT: Inhabitable types:
+;; CHECK-NEXT:
+;; CHECK-NEXT: Built 20 types:
+;; CHECK-NEXT: (type $0 (type $0 (struct)))
+;; CHECK-NEXT: (rec
+;; CHECK-NEXT:  (type $1 (type $1 (struct)))
+;; CHECK-NEXT:  (type $2 (type $2 (array i16)))
+;; CHECK-NEXT:  (type $3 (type $3 (func)))
+;; CHECK-NEXT:  (type $4 (type $4 (func (param (ref $5) i32 i64 f64 f64 (ref eq) v128) (result i64))))
+;; CHECK-NEXT:  (type $5 (type $5 (array (mut i32))))
+;; CHECK-NEXT: )
+;; CHECK-NEXT: (rec
+;; CHECK-NEXT:  (type $6 (type $6 (array (mut funcref))))
+;; CHECK-NEXT:  (type $7 (type $7 (func (param f64) (result i64))))
+;; CHECK-NEXT: )
+;; CHECK-NEXT: (rec
+;; CHECK-NEXT:  (type $8 (type $8 (struct_subtype (field f64 (ref $2) f64 (mut (ref null $9))) $1)))
+;; CHECK-NEXT:  (type $9 (type $9 (func_subtype (param (ref array) i32 i64 f64 f64 anyref v128) (result i64) $4)))
+;; CHECK-NEXT: )
+;; CHECK-NEXT: (rec
+;; CHECK-NEXT:  (type $10 (type $10 (func_subtype (param (ref eq) i32 i64 f64 f64 anyref v128) (result i64) $9)))
+;; CHECK-NEXT:  (type $11 (type $11 (array_subtype (mut funcref) $6)))
+;; CHECK-NEXT:  (type $12 (type $12 (array nullref)))
+;; CHECK-NEXT:  (type $13 none)
+;; CHECK-NEXT:  (type $14 (type $14 (array (ref $6))))
+;; CHECK-NEXT:  (type $15 (type $15 (array i32)))
+;; CHECK-NEXT: )
+;; CHECK-NEXT: (rec
+;; CHECK-NEXT:  (type $16 (type $16 (array_subtype nullref $12)))
```

Comment:
This `nullref` is new.

Reply:
Ah, I just compared the nominal one manually, and saw no diff... but yes, this looks fixed up. sgtm